### PR TITLE
add new posts

### PR DESCRIPTION
**Pull request by kishoian**

### CODE REVIEW

File: content/attacks/posts/2018-11-AurumCoin-Cryptopia [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+date: 2018-11
+custodians: AurumCoin, Cryptopia
+categories: 51%
+title: AurumCoin Users Face Losses in a 51% Attack
```
Comment (jhirschkorn) [inline content/attacks/posts/2018-11-AurumCoin-Cryptopia:5]: add quotations to the title

File: content/attacks/posts/2018-11-AurumCoin-Cryptopia [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+## Summary
+
+In November 2018, AurumCoin (AU), a relatively lesser-known cryptocurrency, was hit by a 51% attack. During the attack, an anonymous entity was able to gain control of a majority of the network's hash rate, which enabled them to alter the blockchain ledger and perform double-spend transactions. The attack resulted in the loss of 15,752.26 AurumCoins from the Cryptopia exchange, which was valued at approximately $500,000 at the time.
+
+## Attackers
+
+The attackers behind the 51% attack on AurumCoin remain unidentified. They were able to accumulate a majority of the network’s hash power, thereby taking control of the AurumCoin blockchain. This control enabled them to alter the blockchain ledger to their advantage and execute double-spend transactions.
```
Comment (jhirschkorn) [inline content/attacks/posts/2018-11-AurumCoin-Cryptopia:14]: dont simply restate info from the summary. add info about the attackers or just leave the comment that they are unknown.

File: content/attacks/posts/2018-11-AurumCoin-Cryptopia [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+## Losses
+
+The attack led to the double-spending of 15,752.26 AurumCoins, which were withdrawn from the Cryptopia exchange. At the time, these AurumCoins were valued at approximately $500,000. The loss had a significant impact on the AurumCoin community and raised questions about the security measures in place.
+
+## Timeline
+
+- **November 2018:** AurumCoin network experiences a sudden increase in hash rate, indicating an attempted takeover of the network.
+- **November 2018:** The 51% attack occurs as the anonymous attacker gains control over the majority of the network’s hash rate, which enables them to alter the blockchain ledger and execute double-spend transactions.
+- **November 2018:** Cryptopia exchange reports a loss of 15,752.26 AurumCoins valued at approximately $500,000 due to the attack.
+- **Following Days:** The cryptocurrency community discusses the implications of the attack, with many raising concerns regarding the security of smaller cryptocurrencies like AurumCoin.
```
Comment (jhirschkorn) [inline content/attacks/posts/2018-11-AurumCoin-Cryptopia:25]: delete the "following days" line. it's unnecessary.

**Review by jhirschkorn — CHANGES_REQUESTED** [review]

**Review by jhirschkorn — CHANGES_REQUESTED** [review]

**Review by jhirschkorn — APPROVED** [review]

**Comment by jhirschkorn** [comment]
can you add links to sources to address the following issues from the bot:

> AurumCoin network experienced a sudden increase in hash rate in November 2018. ❌
> The claim is false. The search results indicate that AurumCoin was hit by a 51% attack in November 2018, resulting in a loss of funds for the Cryptopia exchange. A 51% attack means that a hacker takes control over a crypto network with more than 51% of its hash rate. This means that the hash rate of the network did not increase suddenly, but rather was taken over by the attacker. Therefore, the claim is false.

**Comment by kishoian** [comment]
I already deleted this part about increasing hash rate

**Pull request merged**